### PR TITLE
LPS-70258 There's no policy for displaying organization's logo

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-70258](https://issues.liferay.com/browse/LPS-70258).<br/><br/>/cc @pei-jung @alee8888 @ChrisKian @dejuknow @patriciajeanperez

### CODE REVIEW

File: modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@ organizations-types-help=Set this JSON object to change the organizations types.
 use-initials-for-default-user-portrait=Use Initials For Default User Portrait
```
Comment (liferay-appsec) [inline modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties:5]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
@topolik this is the change we discussed in our DM

**Comment by lipusz** [comment]
@topolik

**Comment by martamedio** [comment]
@drewbrokke There're some conflicts, could you rebase + resend?
I'll contact @topolik to have a look at it

Thanks!

**Comment by drewbrokke** [comment]
Thanks @martamedio! I just rebased and force-pushed, I'll run CI again.

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
Hey @drewbrokke 

@topolik has reviewed it and he's OK with these changes, I will forward if CI allows it 😉

**Comment by martamedio** [comment]
ci:forward

**Comment by martamedio** [comment]
@drewbrokke please, see: https://github.com/liferay-appsec/liferay-portal/pull/335

**Pull request closed without merging**